cmd/autoheal/healer.go

```diff
@@ -147,8 +147,17 @@ func (b *HealerBuilder) Build() (h *Healer, err error) {
 	h.rulesQueue = workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "rules")
 	h.alertsQueue = workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "alerts")
 
-	// initialize active jobs map:
-	h.activeJobs = new(syncmap.Map)
+	h.jobRunners = map[JobRunnerType]JobRunner{
+		AWX: AWXJobRunner{
+			config: h.config.AWX(),
+			// initialize active jobs map:
+			activeJobs: new(syncmap.Map),
```

Comment:
This does not have to be part this PR, but it would be cleaner if the creator of an AWXJobRunner would not have to supply this map. It could be done in a constructor.

Reply:
Yeah thats a good comment I just noticed that going over the code again :+1: